static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -46,6 +47,8 @@ const fillAdvertSlots = (): Promise<void> => {
         } else {
             displayAds();
         }
+
+        markTime('Commercial: DFP advert slots filled');
```

Comment:
I'd say 'Fired' rather than 'Filled'

Reply:
Why's that? Because DFP may not have filled? I was going by the naming of the file in this case